### ISSUE TITLE
F12 Developer Tools crashes in IE11 Win7 32-bit with "An error has ocurredJSPlugin.3005" 

### ISSUE THREAD

**Issue by bradhenslee** [xdissent/ievms#261]
After opening IE11 on Win7 and starting the F12 Developer Tools, I get an error ("An error has ocurredJSPlugin.3005...") with its stack trace in the DOM Inspector panel, and the panel becomes unresponsive. I couldn't select the error text to copy here.

The issue seems identical to the one described here (see that post for an example of the full error):
http://answers.microsoft.com/en-us/ie/forum/ie11-windows_7/imposible-use-f12-developer-tools-exception-in/09b4f305-e436-4615-a8a5-705b0bc85bbd

As discussed in that thread, the problem can be resolved by manually installing KB3008923, which can be downloaded here:
32 bit: https://www.microsoft.com/en-us/download/confirmation.aspx?id=45134
64 bit: https://www.microsoft.com/en-us/download/confirmation.aspx?id=45154

**Comment by rocketraman** [comment]
Same issue (and resolution) here, thanks for the analysis and link. It would be great if this update was installed automatically on the VM.

**Comment by gabrielecirulli** [comment]
:+1:, I ended up reinstalling all of my VMs thinking it would fix this and then discovered that it wasn't the case before I found out this was the real issue. It'd totally help to have the script install it automatically.

**Comment by xdissent** [comment]
Yeah that's lame. I'm experimenting now to see if we can't just run an automatic windows update on first boot rather than installing a particular patch. If that's overkill I'll certainly consider adding this one though.

**Comment by trkoch** [comment]
@xdissent I agree that automatically installing the updates is a better solution, but could you add a note to the README in the meantime?

**Comment by dennisgaebel** [comment]
Any fix to this?

![screenshot 2015-11-20 09 41 31](https://cloud.githubusercontent.com/assets/13485205/11302396/36c19d10-8f6b-11e5-882f-0e40f5e917ed.png)

**Comment by jiujiteiro** [comment]
@dennisgaebel the fix in the first comment works.

**Comment by obrejla** [comment]
Thanks for the link!!!

**Comment by magi00055** [comment]
i have an hasoccured error 3005

**Comment by magi00055** [comment]
i have an hasoccured error 3005
